package.json

```diff
@@ -52,7 +52,7 @@
         "clean-css": "^4.2.3",
         "codedown": "^2.1.6",
         "commander": "^5.1.0",
-        "eslint": "^7.1.0",
+        "eslint": "^7.2.0",
```

Comment:
This version bump to eslint is off topic for the scope of the current PR. Please revert it, and then once you've done that please also revert the changes you've made to `yarn.lock`